includes/Modules/Analytics.php

```diff
@@ -1273,7 +1274,11 @@ private function get_home_domain() {
 	private function print_tracking_opt_out() {
 		?>
 		<!-- <?php esc_html_e( 'Google Analytics user opt-out added via Site Kit by Google', 'google-site-kit' ); ?> -->
-		<script type="text/javascript">window["_gaUserPrefs"] = { ioo : function() { return true; } }</script>
+		<?php if ( $this->context->is_amp() ) : // TODO: update script type to `text/plain` when supported by AMP plugin. ?>
```

Comment:
Let's remove the comment here per https://github.com/ampproject/amp-wp/issues/4392

Reply:
Yeah, I suppose that is no longer relevant 😄 

Done!